doc/Uplink.xml

```diff
@@ -295,17 +295,17 @@
       <para>The device shall assign any tunneled HTTP or RTSP request the UserLevel that is configured for the configuration without the need of further authentication request headers. </para>
       <section>
         <title>mTLS authentication</title>
-        <para>To signal that a device supports this authentication mechanism, it shall include <literal>mTLS</literal> in the supported <literal>AuthorizationModes</literal> in the GetServiceCapabilities response.</para>
+        <para>Device signaling support for mTLS based authentication shall include <literal>mTLS</literal> in the list of <literal>AuthorizationModes</literal> of GetServiceCapabilities response.</para>
         <para>When using mTLS authentication, the local service shall authenticate itself at the
           remote client using TLS client authentication according to RFC 5246 or subsequent
           specifications.</para>
         <para>To uniquely identify a local service on remote client, it is recommended to have a unique client certificate installed on each local service. For example, CN field or Serial number of installed certificate could be used to uniquely identify the local service.</para>
       </section>
       <section>
         <title>Access token authentication</title>
-        <para>To signal that a device supports this authentication mechanism, it shall include <literal>AccessToken</literal> in the supported <literal>AuthorizationModes</literal> in the GetServiceCapabilities response. It shall also have a value greater than 0 in the <literal>MaxConfigurations</literal> under AuthorizationServer in the GetServiceCapabilities response from the Security service.</para>
-        <para>For the local service to authenticate with the remote service, it shall include the access token retrieved from the configured <literal>AuthorizationServer</literal> in the <literal>Authorization</literal> header when sending the upgrade request.</para> 
-        <para>The remote client shall validate the access token.</para>        
+        <para>Device signaling support for access token based authentication shall include <literal>AccessToken</literal> in the list of AuthorizationModes of GetServiceCapabilities response. Device shall signal value greater than 0 for MaxConfigurations in AuthorizationServer.GetServiceCapabilities response from the Security service.</para>
+        <para>To authenticate with the remote client or service, the local service shall include the access token retrieved from the configured <literal>AuthorizationServer</literal> in the Authorization header when sending the upgrade request.</para>
+        <para>The remote client shall validate the access token.</para>
```

Comment:
@jflevesque-genetec 

Just one small question, Isn't this requirement written from client or remote service perspective - which seems a bit contrary to how we always write requirements from device (now acting as client) ? 

In that sense, we can only mandate that device shall send the access token in the upgrade request but remote service validation could be seen as out of scope of our requirement set?

Reply:
I mean, we could validate with a negative test (which I think we don't usually do, ex. send invalid token to see if we receive error from service).

If we don't mention, its still an implicit requirement for the client.

Reply:
May be we can remove this requirement from technical spec and move to Profile-V as mandatory client requirement, while sending the token in its upgrade request could be device mandatory?

Even if we dont include device (acting as client) requirement in the profile spec,  its expected to implemented as per technical spec and can have a conformance test to validate that.